.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - main
+
+name: Run Release Please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: simple
+          command: manifest
+          token: ${{secrets.TOKEN_PUBLISH}}
```

Comment:
I had to drop this (it then defaults to `secrets.GITHUB_TOKEN`) for the Android flow to work.
```suggestion
```